source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -315,6 +329,17 @@ class NiSyncDriverApiTest : public ::testing::Test {
   std::unique_ptr<::grpc::Server> server_;
 };
 
+TEST_F(NiSyncDriverApiTest, RevisionQuery)
```

Comment:
Can you update the test name to reflect the assertions? Something like `RevisionQuery_ReturnsNonEmptyRevisions`.